packages/patternfly-4/plugins/gatsby-transformer-react-docgen-typescript/gatsby-node.js

```diff
@@ -1,11 +1,13 @@
 const reactDocgen = require('react-docgen');
 
 function isSource(node) {
+  console.log('node', node.relativePath)
   if (
     !node ||
-    node.relativePath.indexOf('/example') !== -1 ||
-    node.relativePath.indexOf('.docs') !== -1 ||
-    node.relativePath.indexOf('.md') !== -1
+    node.relativePath.indexOf('/example') >= 0 ||
+    node.relativePath.indexOf('.docs') >= 0 ||
+    node.relativePath.indexOf('.md') >= 0 ||
+    node.relativePath.indexOf('patternfly-3') >= 0
```

Comment:
I like the suffix addition, thanks! Curious about the changes in example-styles.css and this gatsby-node.js though, are they related?

Reply:
Gatsby was pulling in Patternfly 3 in the markdown for some reason so it was showing both as having the same id and failing axe. I asked Zack to take a look at it, and he actually added that fix and told me it was due to the way he originally wrote it. I believe it just tells the markdown to ignore Patternfly 3 here. I am noticing that I accidentally included the console.log though so I'll take that out!